packages/utils/odsp-doclib-utils/package.json

```diff
@@ -11,31 +11,32 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/odsp-doclib-utils-public.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
-				"types": "./dist/odsp-doclib-utils-public.d.ts",
+				"types": "./dist/index.d.ts",
```

Comment:
If we already have public in place, don't roll it back to "internal".

Reply:
Oops.  Thanks.